react/game/src/app/screens/Profile/components/ProfileEdit/components/ProfileEditForm/layout.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import formNames from '@constants/formNames';
+import Button from '@components/Form/Button';
+
+import styles from './styles.scss';
+
+function ProfileEditFormLayout({
+  handleSubmit,
+  pristine,
+  submitting,
+  errorMessage,
+  successMessage,
+  activateEditProfile,
+  formFields
+}) {
+  const { btnTooltips, btnTitles } = formNames.EDIT_PROFILE;
+  return (
+    <div className={styles.editProfileFormContainer}>
+      <form className={styles.editProfileForm} onSubmit={handleSubmit}>
+        {formFields.map(field => (
```

Comment:
⚠️ Remember about map arrow functions in render!

Reply:
Sorry, sorry!!